catalog/transformer/catalog-transformer-streaming-impl/pom.xml

```diff
@@ -97,9 +102,14 @@
                             gml-v_3_1_1-schema,
                             jaxb2-basics-runtime,
                             catalog-transformer-streaming-lib,
-                            platform-util
+                            platform-util,
+                            org.apache.felix.utils
                         </Embed-Dependency>
-                        <Export-Package />
+                        <Export-Package/>
+                        <Import-Package>
+                            !org.apache.felix.utils.collections,
```

Comment:
This is a smell, we should be managing exclusions through the maven dependencies instead. Otherwise you wind up with different behavior at build and at run.